packages/vite/src/node/plugins/esbuild.ts

```diff
@@ -98,17 +99,9 @@ export async function transformWithEsbuild(
           compilerOptionsForFile[field] = loadedCompilerOptions[field]
         }
       }
-
-      // align with TypeScript 4.3
-      // https://github.com/microsoft/TypeScript/pull/42663
-      if (loadedCompilerOptions.target?.toLowerCase() === 'esnext') {
-        compilerOptionsForFile.useDefineForClassFields =
-          loadedCompilerOptions.useDefineForClassFields ?? true
-      }
     }
 
     tsconfigRaw = {
-      ...tsconfigRaw,
```

Comment:
Wait, I mistakenly deleted this line… 😂
@patak-dev Could you push a commit to the main branch to fix up this one?

Reply:
Uh 😅
Please create a PR and I'll merge it. Or Ill push a commit when Im not in mobile

Reply:
Done in https://github.com/vitejs/vite/commit/7e6a2c8f7d140186576b9492639f462b3eb99e12